Physics3D/geometry/triangleMesh.cpp

```diff
@@ -177,31 +181,82 @@ TriangleMesh::TriangleMesh(UniqueAlignedPointer<float>&& vertices, UniqueAligned
 TriangleMesh::TriangleMesh(int vertexCount, int triangleCount, const Vec3f* vertices, const Triangle* triangles) :
 	MeshPrototype(vertexCount, triangleCount) {
 
-	size_t vertexOffset = getOffset(vertexCount);
+        size_t vertexOffset = 8;//getOffset(vertexCount);
 
+	
+	//printf("vertex_count = ori:%d, now: %lu\ntriangle_count = ori:%d, now: %lu\n", vertexCount, getOffset(vertexCount) * 3, triangleCount, getOffset(triangleCount) * 3),
+	
+	//printf("TriangleMesh::TriangleMesh->vertex_offset = %lu\n", vertexOffset);
+	
+	//printf("TriangleMesh::TriangleMesh->vertexCount = %d\n", vertexCount);
+	
```

Comment:
This file is filled with commented out debug prints, remove. Also inconsistent indentation